packages/editor-ui/src/composables/useRunWorkflow.ts

```diff
@@ -259,7 +259,7 @@ export function useRunWorkflow(useRunWorkflowOpts: { router: ReturnType<typeof u
 				executedNode,
 				data: {
 					resultData: {
-						runData: newRunData ?? {},
+						runData: startRunData.runData ?? {},
```

Comment:
Fixes that previous run data disappears. This should be the same run data that is sent to the BE. 

`startRunData.runData` is either the run data the FE pruned (if the old flow is used) or the full run data from before (if the new flow is used).